main.py

```diff
@@ -0,0 +1,20 @@
+import logging
+from logging import config
+
+logging.config.fileConfig('logging.conf')
+logger = logging.getLogger(__name__)
+
+import asyncio
+import os
+from commons import JenkinsJob
+from IPython.display import HTML, display
+
+async def main():
+    user, password = os.environ['AUTH'].split(":")
+    job = JenkinsJob("jenkins.mesosphere.com/service/jenkins/view/Marathon/job/", "marathon-sandbox/job/marathon-loop-1.7", user, password)
```

Comment:
did you mean to hard-code marathon-loop-1.7 here?

Reply:
No. This was just for testing purposes. I used the Jupyter notebook.